tools/ci_connector_ops/ci_connector_ops/qa_checks.py

```diff
@@ -93,9 +126,22 @@ def check_connector_https_url_only(connector: Connector) -> bool:
     Returns:
         bool: Wether the connector code contains only https url.
     """
-    # TODO implement
+    files_with_http_url = set()
+    for filename, line in read_all_files_in_directory(
```

Comment:
@alafanechere I think a small improvement here would be to omit lines that are comments.

```python
def is_comment(line, file_path):
    language_comments = {
        ".py": "#",
        ".yml": "#",
        ".yaml": "#",
        ".java": "//",
        ".md": "<!--",
    }

    denote_comment = language_comments.get(file_path)
    if not denote_comment:
        return False

    trimmed_line = line.lstrip()    
    return trimmed_line.startswith(denote_comment)
```